hub/src/server/config.ts

```diff
@@ -54,7 +78,7 @@ function getConfigEnv(envVars) {
           throw new Error(`Passed a non-number input to: ${envVar}`)
         }
       } else if (parseLists.indexOf(name) >= 0) {
-        configEnv[name] = configEnv[name].split(',').map(x => x.trim())
+        configEnv[name] = (<string>configEnv[name]).split(',').map(x => x.trim())
```

Comment:
I'm unfamiliar with this syntax.  Is this a typecast?

Reply:
Yep!